tests/modules/images/webp-uploads/helper-tests.php

```diff
@@ -231,4 +231,37 @@ function () {
 		$this->assertWPError( $result );
 		$this->assertSame( 'image_mime_type_not_supported', $result->get_error_code() );
 	}
+
+	/**
+	 * Create an image with the filter webp_uploads_pre_generate_additional_image_source added.
+	 *
+	 * @test
+	 */
+	public function it_should_create_an_image_with_filter_webp_uploads_pre_generate_additional_image_source() {
+		$attachment_id = $this->factory->attachment->create_upload_object( TESTS_PLUGIN_DIR . '/tests/testdata/modules/images/car.jpeg' );
+
+		add_filter(
```

Comment:
We need to call `remove_all_filters( 'webp_uploads_pre_generate_additional_image_source' )` before adding a new filter to make sure that no other hooks are registered for this filter.

Reply:
@eugene-manuilov Thanks, I have updated the changes.